lib/eigensolve_quda.cpp

```diff
@@ -395,13 +416,13 @@ namespace quda
       for (int j = 0; j < size; j++) {
         auto cnorm = H[i*size + j];
         if (j != i) {
-          if (abs(cnorm) > epsilon) {
+          if (abs(cnorm) > 5.0 * epsilon) {
```

Comment:
Why this arbitrary factor of 5.0?

Reply:
Just using `epsilon=2e-16` would sometimes fail. I chose 5 to be safe after a few random iterations.